pom.xml

```
@@ -33,7 +33,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <Implementation-Version>${project.version}</Implementation-Version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <selenium.version>4.14.0</selenium.version>
+        <selenium.version>4.16.0</selenium.version>
```

```suggestion
        <selenium.version>4.16.1</selenium.version>
```

4.16.0 is broken: https://github.com/SeleniumHQ/selenium/blob/trunk/java/CHANGELOG